app/Http/Controllers/CircleController.php

```diff
@@ -3,6 +3,7 @@
 namespace App\Http\Controllers;
```

Comment:
is there an auto-formatter plugin for PHP, like prettier for JS? might be good to just standardize formatting around something like that.

Reply:
Not something I need you to figure out in this PR of course; just thinking out loud. do you have anything set up in your IDE that's making these formatting changes automatically?

Reply:
there is though just that it requires to trigger manually with hotkeys to autoformat , will look into that